ext/datadog_cov/datadog_cov.c

```diff
@@ -159,6 +120,12 @@ static VALUE dd_cov_start(VALUE self)
   return self;
```

Comment:
:arrow_up: Also, now that we're moving away from line coverage, maybe it would be a good idea to revisit the use of `RUBY_EVENT_LINE`? E.g. perhaps a combination of coarser-grained events (call, b_call, fiber_switch, etc) would be an overall performance win?

Reply:
Using RUBY_EVENT_CALL increased overhead twice: I guess this event is not optimized as good as RUBY_EVENMT_LINE